components/avatar/avatar.vue

```diff
@@ -7,29 +7,36 @@
       AVATAR_SIZE_MODIFIERS[size],
       AVATAR_COLOR_MODIFIERS[color],
       avatarClass,
+      'd-ps-relative',
     ]"
     data-qa="dt-avatar"
   >
     <!-- @slot Slot for avatar content -->
     <slot />
+    <dt-presence
+      v-if="presence"
+      :presence="presence"
+      class="d-ps-absolute d-b0 d-r0"
+    />
```

Comment:
Do we need different sizes of presence for the different sizes of avatar? It looks a bit off at large and small sizes.

Reply:
Good point - I haven't considered Avatar when integrating this. 

@francisrupert I guess that's kind of a design question as well - do you think the presence circle should adjust its size depending on the Avatar's size?

Here's where you can see this happening: https://vue.dialpad.design/deploy-previews/pr-586/?path=/docs/components-avatar--presence

Reply:
The _positioning_ of Presence should indeed be very slightly different for each size. Presence size remains the same.

Use the alignment from [this Figma file](https://www.figma.com/file/Vw71A2boup3Gb3PqUQTSiu/Avatar-%2B-Presence?node-id=1%3A2&viewport=1966%2C1826%2C3.85&t=e8mYJk5515vSJQbQ-11). Let me know if you've any questions.

<img width="241" alt="image" src="https://user-images.githubusercontent.com/1165933/202085210-a1a4081b-c6e0-44ed-b7a3-c75c4a622db2.png">

Reply:
@francisrupert great thanks! It looks like we're moving the presence slightly toward the center as the Avatar gets larger, right?

@braddialpad Here's what I'm planning to do for this:

- I'll open a PR against `dialtone` to add the `position: relative` css attribute to `d-avatar`.
- I'll close this PR and open a new one updating the stories and using `presence` as suggested by Francis above. From what it looks like, we'll have to update the absolute position values of `top` and `right` depending on the avatar size.

Reply:
@modialpad,

Righto. Up in and in as avatar size increases. Positioning from `bottom` and `right` will work best; probably best to do so contextually in the CSS rather than use positioning CSS Utilities.

Note the actual position value in CSS will vary from how Figma treats it. That is, Figma doesn't consider the gray outline part of the box model while your current HTML/CSS does. So the fun is to zoom in and pixel-push as appropriate to align to Figma visual.